client_app/src/styles/shelter/MoreDetailsBox.css

```diff
@@ -0,0 +1,75 @@
+.modal-overlay {
+    background-color: rgba(0, 0, 0, 0.75);
+    display: flex;
+    align-items: center;
+    justify-content: center;
+  }
+  
+  .modal-content {
+    background: white;
+    padding: 20px;
+    border-radius: 8px;
+    width: 400px;
+    max-width: 100%;
+  }
+  /*ask about colors*/
+  .shift-details-card {
+    background-color: #f9f9f9;
+    border-radius: 10px;
```

Comment:
now to make these work you need to name the actual components inside of the MoreDetailsBox.js. I connected the css sheet

Reply:
Then you can also delete the open modal button styling as I added the connectivity to yours.

Reply:
So right now I can't see much of the styling but let me know when you change this and then I will relook at it

For now for padding and margins (unless they're small margins, use % instead of px). The border-radius px is okay